src/main/java/net/neoforged/neoforge/event/entity/player/PlayerEvent.java

```diff
@@ -444,24 +444,6 @@ public PlayerLoggedOutEvent(Player player) {
         }
     }
 
-    public static class PlayerRespawnEvent extends PlayerEvent {
-        private final boolean endConquered;
-
-        public PlayerRespawnEvent(Player player, boolean endConquered) {
-            super(player);
-            this.endConquered = endConquered;
-        }
-
-        /**
-         * Did this respawn event come from the player conquering the end?
-         * 
-         * @return if this respawn was because the player conquered the end
-         */
-        public boolean isEndConquered() {
-            return this.endConquered;
-        }
-    }
-
```

Comment:
The usecase of the new event is not sufficient to replace the old event, nor can it ever be.  The old event has the specific ability to modify the newly-constructed `ServerPlayer`.